### PR TITLE
Add pentanomial to live elo table.

### PR DESCRIPTION
**Pull request by peregrineshahin**

### CODE REVIEW

File: server/fishtest/static/js/live_elo.js [hunk 1]
```diff
@@ -161,6 +161,10 @@ async function follow_live(test_id) {
     ${items.args.sprt.state ? `(${items.args.sprt.state})` : ""}
   `;
 
+    const pentanomial = items.results?.pentanomial || [];
+    ptnml = `[${pentanomial.slice(0, 5).join(", ")}]`;
+    document.getElementById("pentanomial").textContent = pntml;
```
Comment (ppigazzini) [inline server/fishtest/static/js/live_elo.js:166]: typo
```diff
-    ptnml = `[${pentanomial.slice(0, 5).join(", ")}]`;
-    document.getElementById("pentanomial").textContent = pntml;
+    pntnml = `[${pentanomial.slice(0, 5).join(", ")}]`;
+    document.getElementById("pentanomial").textContent = pntnml;
```

Comment (peregrineshahin) [reply server/fishtest/static/js/live_elo.js]: Fixed.

**Review by ppigazzini — APPROVED** [review]
LGTM on DEV

**Comment by ppigazzini** [comment]
PROD updated, thank you @peregrineshahin :)

**Pull request merged**